input/templates/my-account-edit-address.hbs

```diff
@@ -22,19 +22,19 @@
               </div>
               <div id="my-account-desktop-content" class="col-sm-9">
                 <div class="address-book-content edit-address-book-content">
-                  <form id="form-edit-my-address-book" action="{{@root.meta._links.editMyAddressBook.href}}" method="POST">
+                  <form id="form-edit-my-address-book" action="{{@root.meta._links.myAddressBookEditAddressSubmit.href}}" method="POST">
                     <input type="hidden" name="csrfToken" value="{{@root.meta.csrfToken}}"/>
-                    <input type="hidden" name="id" value="{{content.address.id}}"/>
+                    <input type="hidden" name="id" value="{{content.addressForm.id}}"/>
```

Comment:
This information can go inside the `addressForm` without problems, so we can avoid passing the `address` info as well.